pkg/github/runtime_release.go

```
@@ -17,6 +18,18 @@ const (
 	runtimeRepo  = "spiceai"
 )
 
+func CheckForLatestVersion() {
```

As this is CLI specific it should probably go in something like `pkg/cli/version.go`

---

It should also return the error, which should be checked by the caller. I.e.

```go
func checkForLatestVersion() error {
```

---

Got your point. Will do the changes now